src/plugins/intel_gna/src/backend/gna_limitations.hpp

```diff
@@ -263,10 +263,12 @@ class Limitations {
     void check_all_ops_supported(const std::shared_ptr<ov::Model>& model,
                                  const InferenceEngine::Precision gna_precision);
 
-    bool use_only_16bit_convolution_weights() const;
     bool is_crop_affined_offset(size_t numberOfElements) const;
     bool is_aligned(size_t addr) const;
+    bool has_internal_memory_buffers() const;
+    bool does_gna_lib_reserve_scratch() const;
```

Comment:
```suggestion
    bool gna_lib_reserves_scratch() const;
```

Reply:
This doesn't follow the convention of bool function naming - not a question, but a statement. Could also be scratch_reserved_by_gna_lib.

Reply:
ok